src/main/java/DukeManager.java

```diff
@@ -0,0 +1,131 @@
+public class DukeManager {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+    private static final String ADD_CONFIRMATION_MESSAGE = "Got it. I've added this task:";
+    private static Task[] inputs = new Task[100];
+    private static int index = 0;
+
+    /**
+     * Returns a string with formatted user inputs.
+     *
+     * @param inputs list of user input.
+     * @return Formatted string to print
+     */
+    public static String formatList(Task[] inputs) {
+        String formattedString = HORIZONTAL_LINE;
+        for (int i = 0; i < inputs.length; i++) {
+            if (inputs[i] == null) {
+                break;
+            }
+            formattedString += (i+1) + "." + inputs[i].toString() + "\n";
```

Comment:
Perhaps the operators can be separated by whitespace?
```suggestion
            formattedString += (i + 1) + "." + inputs[i].toString() + "\n";
```

> Operators should be surrounded by a space character.

https://se-education.org/guides/conventions/java/index.html#layout